picard/util/pipe.py

```diff
@@ -243,15 +243,23 @@ def _sender(self, message: str) -> bool:
             return False
 
         try:
-            with open(self.path, 'w') as fifo:
+            with open(self.path, 'a') as fifo:
                 fifo.write(message)
+            log.debug("sent successfully: %r", message)
```

Comment:
For extra debug, if you really want to keep that, define a boolean constant (like `DEBUG_PIPE`) and use tests:

```python
if DEBUG_PIPE:
    log.debug("sent successfully: %r", message)
```

As is, that's far too much debug output.

Reply:
So you're saying to remove sent/read/received debug output, right?

Reply:
Yeah that's simply too much